opsramp/resource_groups.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#
+# A minimal Python language binding for the OpsRamp REST API.
+#
+# resources.py
+# Resource classes.
+#
+# (c) Copyright 2020-2021 Hewlett Packard Enterprise Development LP
```

Comment:
This should be 2022

Reply:
Do we need to apply the copyright-check CI job to this repo?

Reply:
It's slightly complicated because this is an open-source project which means there can be contributions by non-HPE people and those changes would not be copyright HPE. Maybe if it could check if the submitter has an hpe.com email address from the env variables or something.